system/web/context/ExceptionBean.cfc

```diff
@@ -41,6 +41,20 @@ component accessors="true"{
 		}
 		variables.extraMessage 	= arguments.extraMessage;
 		variables.extraInfo 	= arguments.extraInfo;
+		
+		try {
+			if(structIsEmpty(variables.exceptionStruct) && isObject(arguments.errorStruct)){
```

Comment:
Hi Luis, 
it was originally with duplicate( arguments.errorStruct ) but changed later on. Brad had some issue so he can explain why he changed. At-least its broken/empty detail,message in error report page for CF10 + 11